src/components/ClusterSpinUp.js

```diff
@@ -1,17 +1,19 @@
 import React, {Component} from 'react';
 
 class ClusterSpinUp extends Component {  
-    clusterLinks = (url) =>   {
+    getUrlVars = (vars) => {
         window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
-        url[key] = value;
```

Comment:
This is a very C way of doing things. In JS I would suggest using immutable objects and returning a new value instead

Reply:
Presumably the qs package will make this redundant